libindy/src/services/wallet/default.rs

```diff
@@ -280,7 +289,7 @@ impl From<rusqlcipher::Error> for WalletError {
     fn from(err: rusqlcipher::Error) -> WalletError {
         match err {
             rusqlcipher::Error::QueryReturnedNoRows => WalletError::NotFound(format!("Wallet record is not found: {}", err.description())),
-            _ => WalletError::CommonError(CommonError::InvalidState(format!("Unexpected SQLite error: {}", err.description())))
+            _ => WalletError::SecurityError(format!("Unexpected SQLite error: {}", err.description()))
```

Comment:
Can we stay InvalidState error for some unexpected errors and catch specific error from rusqlciphter::Error as Security error?